static/app/components/replays/player/scrubber.tsx

```diff
@@ -63,11 +65,21 @@ function Scrubber({className, showZoomIndicators = false}: Props) {
         {currentHoverTime ? (
           <div>
             <TimelineTooltip
-              labelText={formatDuration({
-                duration: [currentHoverTime, 'ms'],
-                precision: 'ms',
-                style: 'hh:mm:ss.sss',
-              })}
+              labelText={
+                timestampType === 'absolute'
+                  ? getFormattedDate(
+                      startTimestamp + currentHoverTime,
+                      shouldUse24Hours() ? 'HH:mm:ss.SSS' : 'hh:mm:ss.SSS',
+                      {
+                        local: true,
```

Comment:
Is startTimeStamp already in the right timezone?

Reply:
all the timestamps we are getting are in milliseconds since epoch, eg. 1725993813532 so we only have to worry about timezones when we are formatting the times

Reply:
Yeah, so should we always use local: true here or switch it based on the timezone?

Reply:
local is whether or not we use the timezone or not (uses UTC if false). `local` isn't a great name for that imo